bin/dev_stop.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "Stopping Argo"
+if [ -e tmp/pids/server.pid ]; then
+    kill -9 $(cat tmp/pids/server.pid)
+    rm tmp/pids/server.pid
+    echo "Stopped Argo server"
+fi
+if [ -e tmp/pids/delayed_job.pid ]; then
+    kill -9 $(cat tmp/pids/delayed_job.pid)
+    rm tmp/pids/delayed_job.pid
+    echo "Stopped delayed_job"
+fi
```

Comment:
In a 'development' env, it's always possible that something is totally screwy and you will need the `kill -9` to shutdown.  In general, it is a good idea to allow a graceful stop.  Perhaps the majority of the time it will be OK without the `-9`, but then the `rm` on the pids should be conditional on successful shutdown.  So, a heavy hammer is what we have here and your very welcome to modify it into a rubber mallet.